src/stores/notifications/SpaceNotificationState.ts

```diff
@@ -74,7 +76,15 @@ export class SpaceNotificationState extends NotificationState {
 
         this._count = 0;
         this._color = NotificationColor.None;
-        for (const state of Object.values(this.states)) {
+        for (const [roomId, state] of Object.entries(this.states)) {
+            const roomTags = RoomListStore.instance.getTagsForRoom(this.rooms.find(r => r.roomId === roomId));
+
+            // We ignore unreads in LowPriority rooms, see https://github.com/vector-im/element-web/issues/16836
+            if (
+                Object.keys(roomTags).includes(DefaultTagID.LowPriority) &&
+                state.color === NotificationColor.Bold
+            ) continue;
```

Comment:
This looks wrong, roomTags is a string array, not an object

![image](https://user-images.githubusercontent.com/2403652/128337973-fcf0af3f-7995-4e9a-a636-cd7b255257cc.png)


```suggestion
            if (
                roomTags.includes(DefaultTagID.LowPriority) &&
                state.color === NotificationColor.Bold
            ) continue;
```

Reply:
Sorry about that